scripts/build_gnome_supp.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I think `$(dirname "$0")` should do the same.

Reply:
I took it from this: https://stackoverflow.com/a/246128/409102. I guess, there was a reason, why it's such complex. I think it's something with symlinks, and.. I don't know,.. some magic?

Reply:
You could feed the output through `readlink -f`, but I don't think this is really necessary: you don't really care about the absolute path here?